doc/api/process.md

```diff
@@ -745,7 +745,7 @@ process.exit(1);
 The shell that executed Node.js should see the exit code as `1`.
 
 It is important to note that calling `process.exit()` will force the process to
-exit as quickly as possible *even if there are still asynchronous operations 
```

Comment:
Yikes.  Looks like my text editor stripped trailing spaces; I'm guessing they're important, so I'll fix the commit real quick.